doc/src/stdlib/base.md

```diff
@@ -110,6 +110,7 @@ Base.isleaftype
 Base.typejoin
 Base.typeintersect
 Base.Val
+Core.Void
```

Comment:
Yeah, but I wondered whether it makes any difference since it's reexported from `Base`.